chapters/en/unit2/cnns/yolo.mdx

```diff
@@ -1,46 +1,105 @@
-# YOLO
+# YOLO 
+
+## A short introduction to Object Detection
+
+Convolutional Neural Networks has taken a big step towards the image classifcation problem. But there remains
+another big task to solve. Object detection. Object detection not only requires to categorize the object from the image but also to accurately predict the location (in this case the co-ordinates of the bounding boxes of the object) from the image. Here came the big breakthrough of YOLO. Before delving into the depth of YOLO, lets get through the history of object detection algorithms using CNN.
+
+
+### RCNN , Fast RCNN , Faster RCNN
+
+#### RCNN
+RCNN is the simplest way possible to use Convolutional neural network for Object detection. to be as simple as possible , the basic idea is to 
+detect a "region" and then use CNN to classify the region . So this is a multi-step process . Based on this idea the RCNN paper was shared in 2012[1]
+
+The RCNN uses following steps , 
+
+- Use Selective Search algorithm to select a region. 
+- Use CNN based classifier to classify an object from the region. 
+
+For training purpose, the paper proposed following steps 
+
+- Make a dataset of regions detected from the Object detection dataset. 
+- Fine tune Alexnet model on the regions dataset . 
+- Then use the fine tuned model on the object detection dataset . 
+
+The following is a basic pipeline of RCNN 
+![rcnn](https://huggingface.co/datasets/hf-vision/course-assets/blob/main/RCNN.png)
+
+
+#### Fast RCNN 
+
+Fast RCNN is nothing but an improvization over RCNN. They added these three improvements
```

Comment:
Instead of 'Fast RCNN is nothing but an improvisation over RCNN.' you can mention FastRCNNs focused on improving over RCNN in terms of training and inference speed. Specifically, They added.